velox/exec/HashProbe.cpp

```diff
@@ -1440,21 +1488,20 @@ int32_t HashProbe::evalFilter(int32_t numRows) {
     static const char* kPassed = "passed";
 
     if (nullAware_) {
-      leftSemiProjectIsNull_.resize(numRows);
-      leftSemiProjectIsNull_.clearAll();
+      leftSemiProjectIsNull_.resize(numRows + offset, false);
```

Comment:
We can initialize it once outside the loop. Each loop will only write into non-overlapping subranges here.

Reply:
I see, updated.